lib/restrictions.js

```diff
@@ -73,6 +73,25 @@ export async function insert({restriction} = {}) {
   return record;
 }
 
+/**
+ * Inserts multiple restrictions into database.
+ *
+ * @param {object} options - The options to use.
+ * @param {Array} options.restrictions - An array of restrictions.
+ * @returns {Promise} - Settles once the operation completes.
```

Comment:
```suggestion
 *
 * @returns {Promise} - Resolves once the operation completes.
```
(blank line before the `@returns` statement).